pyspades/server.py

```diff
@@ -316,14 +316,17 @@ def got_master_connection(self, connection):
     def master_disconnected(self, client=None):
         self.master_connection = None
 
-    def update_master(self):
-        if self.master_connection is None:
-            return
+    def get_player_count(self):
```

Comment:
How about: 
```python
@property
def player_count(self):
```

Reply:
I considered it. However, I don't like it when properties hide the expensiveness of calculations. In this case, I decided to make it a function since it doesn't just access the player count, but calculate it.